samples/gamma/README.md

```diff
@@ -0,0 +1,13 @@
+gamma
+=====
+
+Example of using GPU gamma ramps for tonemapping and effects.
+
+By not setting a linear ramp, you are typically reducing the amount of colors
+that you can output to the display.
```

Comment:
nit: wrapping

It might be nice to include your suggestion of using this to produce fade effects, and maybe some resources here on how to go about calculating these values (or a note about the links you've provided in the sample code).

Reply:
How would you like to have this wrapped? I did manually wrap at 80 columns.

Reply:
> How would you like to have this wrapped? I did manually wrap at 80 columns.

Looks like you fixed it. The version I reviewed had a sentence wrap randomly at 36 chars.

> I think that people will figure this out themselves.

Fair enough I suppose. Would like to provide more context for newbies, but I guess we don't really do it with other samples so we shouldn't have to start now.

> I'm unable to take a screenshot due to lack of emulation in XQEMU. With this effect not being part of framebuffer I also can't easily dump it from a physical Xbox. A photo of a display isn't nice either, and a photoshopped screenshot also isn't very good.

Let's revisit once we emulate this in XQEMU properly, or if someone can grab a screenshot through a capture card that would be nice too (@ryze119 perhaps?)

> We can probably update the description with more information in the future, when we improve / split the sample (like providing an actual gamma slider for compensating for the users display). For now, this is probably "good enough".

Agreed, don't feel like bikesheding

Reply:
> Looks like you fixed it. The version I reviewed had a sentence wrap randomly at 36 chars.

The old one was wrapped at 80 columns (not automatically, but manually).
I changed it because manually breaking at 80 columns would be annoying for my follow up samples (noticed while working on gamma-fade PR 268).
I just use shorter sentences now and have one sentence per line.